http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -820,6 +820,7 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       // Node selector config: this ensures the app is run on the user's nodepool
       raw"""galaxy.nodeSelector.cloud\.google\.com/gke-nodepool=${nodepoolName.value}""",
       raw"""nfs.nodeSelector.cloud\.google\.com/gke-nodepool=${nodepoolName.value}""",
+      raw"""galaxy.configs.job_conf\.yml.runners.k8s.k8s_node_selector=cloud.google.com/gke-nodepool: ${nodepoolName.value}""",
```

Comment:
Corresponds to https://github.com/galaxyproject/galaxykubeman-helm/blob/master/galaxykubeman/values.yaml#L139

It's a little weird in that the property value is a string with a colon, e.g. `cloud.google.com/gke-nodepool: ${nodepoolName.value}`. I'm going to manually check that it is set correctly in k8s.

Reply:
Is that `space` after `gke-nodepool:` supposed to be there?

Reply:
I think it is, not sure if it should be quoted here though -- will test

Reply:
I think this is actually correct, Galaxy starts up in my fiab and the conf looks correct to me:
```
runners:
      k8s:
        k8s_cleanup_job: onsuccess
        k8s_fs_group_id: "101"
        k8s_galaxy_instance_id: 't08w2e-gxy-rls'
        k8s_job_api_version: batch/v1
        k8s_namespace: 't08w2e-gxy-ns'
        k8s_node_selector: 'cloud.google.com/gke-nodepool: k4173ee2-16cd-42e1-bd85-7f30913f88a7'
        k8s_persistent_volume_claims: |-
          t08w2e-gxy-rls-galaxy-pvc:/galaxy/server/database,t08w2e-gxy-rls-galaxy-cvmfs-gxy-data-pvc:/cvmfs/data.galaxyproject.org,t08w2e-gxy-rls-galaxy-cvmfs-gxy-main-pvc:/cvmfs/main.galaxyproject.org
        k8s_pod_priority_class: 't08w2e-gxy-rls-galaxy-job-priority'
        k8s_pod_retries: "4"
        k8s_pull_policy: IfNotPresent
        k8s_run_as_group_id: "101"
        k8s_run_as_user_id: "101"
        k8s_supplemental_group_id: "101"
        k8s_use_service_account: true
        k8s_walltime_limit: "604800"
        load: galaxy.jobs.runners.kubernetes:KubernetesJobRunner
      local:
        load: galaxy.jobs.runners.local:LocalJobRunner
        workers: 4
```

Reply:
yay 👍

Reply:
However it looks like deletion does not work in this version, the `pre-delete-job` is failing. I pinged Alex and Dannon.